content/3_Silver/Graph_Traversal.mdx

```diff
@@ -151,9 +167,9 @@ static void dfs(int currentNode) {
 <CPPSection>
 
 ```cpp
-const int MAX_N = 1e6;
-vector<int> adj[MAX_N];
-bool visited[MAX_N];
+int n = 6;
+vector<int> adj[n];
+bool visited[n];
```

Comment:
variable length array is not part of c++ standard. Use `const` instead.

Reply:
or just use a vector